children/tests/test_api.py

```diff
@@ -790,6 +793,69 @@ def test_child_enrolment_count(
     assert executed["data"]["child"]["enrolmentCount"] == expected_count
 
 
+@freeze_time("2021-02-02T12:00:00Z")
+@pytest.mark.parametrize(
+    "year_delta,expected_count",
+    [(-2, 0), (-1, 1), (0, 2), (None, 2), (1, 1), (2, 0)],
+)
+def test_enrolment_count_with_ticket_system_passwords(
```

Comment:
`test_enrolment_count_with_ticket_system_passwords` could be renamed to `test_child_enrolment_count_with_ticket_system_passwords` to keep it more in line with the other namings in this file.

Reply:
Taken into account.